src/filters/resize_filter.class.js

```diff
@@ -62,6 +62,9 @@
      * @param {Number} scaleY
      */
     applyTo: function(canvasEl, scaleX, scaleY) {
+      if (scaleX === 1 && scaleY === 1) {
+        return;
+      }
```

Comment:
make filter exit if we want have a bigger image than original.
The resize filter is meant for downscaling and too slow to be used as upscaler. There are no added details